build/UIKit.Xaml/dll/UIKit.Xaml.vcxproj

```diff
@@ -225,18 +225,12 @@
   <ImportGroup Label="ExtensionTargets">
     <Import Project="$(StarboardBasePath)\msvc\sdk-build.targets" />
   </ImportGroup>
-  <ItemGroup>
-    <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)$(XamlGeneratedOutputPath)\**\*.xaml">
-      <DestinationSubDirectory>$(OutputName)\%(RecursiveDir)</DestinationSubDirectory>
-    </ReferenceCopyLocalPaths>
-  </ItemGroup>
-  <Target Name="AddExtraPackageItems" BeforeTargets="GetPackageContents">
+  <Target Name="CopyXbfFiles" BeforeTargets="_CopyFilesMarkedCopyLocal" DependsOnTargets="$(AfterBuildCompileTargets)">
```

Comment:
> <Target Name="CopyXbfFiles" BeforeTargets="_CopyFilesMarkedCopyLocal" DependsOnTargets="$(AfterBuildCompileTargets)"> [](start = 1, length = 118)

So why was this target needed again? The pri files that we package have the xbf data embedded in them right? 

This is *strictly* for functional tests?

If so please add a comment for this target explaining.